Sources/Profile-View.php

```diff
@@ -1886,14 +1886,15 @@ function list_getUserErrors($start, $items_per_page, $sort, $where, $where_vars
  */
 function list_getIPMessageCount($where, $where_vars = array())
 {
-	global $smcFunc;
+	global $smcFunc, $user_info;
 
 	$request = $smcFunc['db_query']('', '
-		SELECT COUNT(*) AS message_count
+		SELECT COUNT(id_msg)
```

Comment:
The most benefit you got by changing the join,
how mutch is the time dif here when you change from id_msg to * ?